README.md

```diff
@@ -64,7 +69,15 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething (thingToDo) {
+  return 7 x thingToDo;
+
+}
+
+function thingToDo (a, b) {
+  return a + b;
+}
+doSomething(thingToDo);
```

Comment:
this is close, but really in line 73, we need to actually invoke the callback. Something like :

```js
function doSomething(thingToDo){
  return 7 x thingToDo(5, 7)
}
```